core/src/ConfirmationModal.html

```diff
@@ -14,7 +14,7 @@ <h3 class="fd-title fd-title--h5">{settings.header}</h3>
         </div>
```

Comment:
in the line above, h3 is used, but fd styles has h2
![image](https://user-images.githubusercontent.com/7690363/108389710-38a03a00-7210-11eb-9c35-4e1ac46b0932.png)